deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -81,39 +83,108 @@ spec:
     $ systemctl status multipathd
     $ multipath -ll
     ```
-    
+
     Important: When configuring Linux multipath devices, verify that the `find_multipaths` parameter in the `multipath.conf` file is disabled.
       - RHEL 7.x: Remove the `find_multipaths yes` string from the `multipath.conf` file.
-    
+
     #### 3. Configure storage system connectivity.
     3.1. Define the hostname of each OpenShift worker node on the relevant storage systems with the valid WWPN or IQN of the node.
-    
+
     3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
-    
+
     3.3. For iSCSI, perform these three steps.
-    
+
     3.3.1. Make sure that the login used to log in to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$> service iscsid restart`.
-    
-    3.3.2. Discover and log into at least two iSCSI targets on the relevant storage
-    systems.
-    
-    ```bash
-    $ iscsiadm -m discoverydb -t st -p ${storage system iSCSI port IP}:3260
-    --discover
-    $ iscsiadm -m node -p ${storage system iSCSI port IP/hostname} --login
+
+    3.3.2. Discover and log into at least two iSCSI targets on the relevant storage systems. (NOTE: Without at least two ports, multipath device will not be created.)
+
+    ```sh
+    $> iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1}:3260 --discover
+    $> iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1} --login
+   
+    $> iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2}:3260 --discover
+    $> iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2} --login
     ```
-    
+
     3.3.3. Verify that the login was successful and display all targets that you logged into. The portal value must be the iSCSI target IP address.
-    
-    ```bash
-    $ iscsiadm -m session --rescan
+
+    ```sh
+    $> iscsiadm -m session --rescan
     Rescanning session [sid: 1, target: {storage system IQN},
-    portal: {storage system iSCSI port IP},{port number}
+    portal: {STORAGE-SYSTEM-iSCSI-PORT-IP1},{port number}
+    portal: {STORAGE-SYSTEM-iSCSI-PORT-IP2},{port number}
+    ```
+
+    End of worker node setup.
+
+
+
+    ## Configuring k8s secret and storage class
```

Comment:
You only need to add Prerequisites in this doc, for further usage, just give a link.

Reply:
why?

why not to give link to the whole prerequisites and userage and make the readme clean?  link to the knowledge center?
also - @rivkap  \ @barakda  what do you think?

Anyway if we put here the prerequisit I think we should provide also detail about the secret ad storage class, so it will be clear that the driver not ready until u do that.

Reply:
I think the main purpose of the guide here is to help the user to install the driver. you are right, it is important to keep it clean and short, otherwise, the user may think it is difficult to install our driver.
But for prerequisit, I think we'd better keep it, so that user don't need to leave ocp before finishing the isntallation.

Reply:
A. So you think secret and storage class should be mentioned as short line with link to the knowledge center?

B. what do u think just state in general that:
prerequisite: 
 1. Install Fibre Channel and iSCSI connectivity rpms, multipath configurations, and configure storage system connectivity. Go to IBM knowlage center for more detail about how to perform the prerequisit -> LINK.
2. Install the operator
3. Install the CR plugin
4. Configuring storage system secrets and relevant storage classes.
   Go to link -> HERE

Reply:
A: yes.
B: 4 is ok to me

Reply:
don't give specific links to sections in the knowledge center.
if you are doing that just mention the chapter name and then the general link (like we've been giving)

Reply:
Today we did a review in TLV team
the decision is to keep prerequisite as is and also add the secret\storage class.

In addition there is a link to the KC at the top of the page.
@27149chen  please approve

Reply:
three "#", so that it is under Prerequisites

Reply:
> Today we did a review in TLV team
> the decision is to keep prerequisite as is and also add the secret\storage class.
> 
> In addition there is a link to the KC at the top of the page.
> @27149chen please approve

ok, If you discussed in a meeting and everyone agree, I'm ok with it. But it must be under Prerequisites section.

Reply:
why it must be under prerequisite?  it practically not a prerequisite(BTW also multipathing is not real prerequisite - you can install the operator with out it).

I prefer to make sure hosts were configure before operator installed, and then next step to install operator and then setup secret and storage class. The user can choose to do sertect and storage class also before. Its mentioned in the description:
"
    ## Configuring k8s secret and storage class
    In order to use the driver, create the relevant storage classes and secrets, as needed. 
    Note: This section can be done also after the operator installation.
"
i think its ok.
please approve

Reply:
why must?
its ok its after prerequisite , because its not. and its mentioned as is:

"    In order to use the driver, create the relevant storage classes and secrets, as needed. 
     Note: This section can be done also after the operator installation."

Reply:
> why must?
> its ok its after prerequisite , because its not. and its mentioned as is:
> 
> " In order to use the driver, create the relevant storage classes and secrets, as needed.
> Note: This section can be done also after the operator installation."

Because you don't mention when the user can start his installation. all the sections are "prerequisite" except this one. You just jump to "post-install" suddenly. the ux is bad. Even you mention that user can start his installation before this section, it is bad too because he must go to somewhere else and go back here again.
That's why I suggest to set it as another "prerequisite". and also, you statement "Note: This section can be done also after the operator installation." is important, this remind the user that he can do it after install if he don't want to do it right now.

Reply:
we mentioned u can do secret\storage class before or after installation
i think its good enough